### PR TITLE
Update welly to 3.1.1.1

### PR DESCRIPTION
**Pull request by carson0321**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/welly.rb [hunk 1]
```diff
@@ -1,20 +1,34 @@
 cask 'welly' do
+  name 'Welly'
+
   if MacOS.version <= :snow_leopard
+    homepage 'https://code.google.com/archive/p/welly'
     version '2.64'
     sha256 '9403a92d74f5e7e02974a0fe8641ab1e320523948a67add596c964b2c4c13cae'
     url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/welly/Welly.v#{version}.zip"
-  else
+  elsif MacOS.version == :lion
```
Comment (reitermarkus) [inline Casks/welly.rb]: Also use `<=` here.

File: Casks/welly.rb [hunk 2]
```diff
@@ -1,20 +1,34 @@
...
+    homepage 'https://code.google.com/archive/p/welly'
     version '2.7'
     sha256 'cb24a26432d8927b1159a1865602c3f30b5190f628167c954e4d6cc723cfcb0f'
     url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/welly/Welly.v#{version}.fix.zip"
-  end
+  else
+    homepage 'https://github.com/clyang/welly'
```
Comment (reitermarkus) [inline Casks/welly.rb]: I think we should just use the current homepage for all versions.

**Comment by commitay** [comment]
This already exists as a separate cask: 
https://github.com/Homebrew/homebrew-cask/blob/master/Casks/clyang-welly.rb

**Comment by carson0321** [comment]
I know the `clyang-welly`. The `welly` has been discontinued. They are the same. I think it's not good to delete the `welly`. However, I always forget to add prefix name, so I wanna merge clyang-welly into welly. I'm not so sure that's a good idea. If no, what should I do?  @commitay  @reitermarkus

**Pull request closed without merging**